core/src/verification/queue/kind.rs

```diff
@@ -96,6 +100,12 @@ pub mod headers {
         }
     }
 
+    impl MemUsage for Header {
```

Comment:
How about implementing MemUsage for the structs who implement rlp::Encodable?

Reply:
```rust
impl<R: Encodable> MemUsage for R {
    fn mem_usage(&self) -> usize {
        rlp::encode(self).to_vec().len()
    }
}
```

Reply:
I think it is good to do so. I'll fix this.

Reply:
Fixed.